apps/prairielearn/src/lib/externalGradingSocket.ts

```diff
@@ -92,7 +92,6 @@ export function connection(socket: Socket) {
       user_id: msg.user_id,
       urlPrefix: msg.url_prefix,
       questionContext: msg.question_context,
-      csrfToken: msg.csrf_token,
       authorizedEdit: msg.authorized_edit,
       renderScorePanels: true,
```

Comment:
This is a bit of a nit, but I was testing this and checking in dev tools what was being sent through the socket, and I noticed that this causes the question footer to be re-rendered even in cases where it doesn't need to be (e.g., instructor question preview). I believe this is existing behaviour, so I won't block on that, I just wanted to point that out. Besides, this will eventually be removed once #10792 is fully merged, since it that case `renderScorePanels` will be set by the router code, not by the socket code.